worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/FileWriter.java

```diff
@@ -308,13 +308,11 @@ public synchronized void destroy(IOException ioException) {
     }
   }
 
-  public void registerDestroyHook(List<FileWriter> fileWriters) {
+  public void registerDestroyHook(ConcurrentHashMap<String, FileWriter> fileWriters) {
```

Comment:
do we need this method?

Reply:
this method is still needed for removing expired filewriter.
expired filewriter will reuse destory method to call hook